awx/ui_next/src/App.jsx

```diff
@@ -65,6 +66,9 @@ function App() {
                         </ProtectedRoute>
                       ))
                       .concat(
+                        <Route exact path="/metrics">
+                          <ObservabilityMetrics />
+                        </Route>,
```

Comment:
```suggestion
```

Please import the screen component and add the route to [ui_next/src/routeConfig.js](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/routeConfig.js#L136-L137)

Reply:
talked about this w/ @AlexSCorey - we actually don't want this in the navbar right now so this is fine as is :+1: